setup.cfg

```diff
@@ -121,6 +121,8 @@ tests =
     pytest-asyncio>=0.17,!=0.23.*
     pytest-cov>=2.8.0
     pytest-xdist>=2
+    pytest-env>=0.6.2
+    pytest-mock>=3.7
     pytest>=6
     testfixtures>=6.11.0
```

Comment:
I think the pytest-env dep should be removed, here's how it is on master:

https://github.com/cylc/cylc-flow/blob/a7cf51ba8522432b13fd9ee148d73c0a2f1122e9/setup.cfg#L121-L124